### ISSUE TITLE
Error after installing "no such table: django_session"

### ISSUE THREAD

**Issue by Starli0n** [allo-/firefox-profilemaker#205]
Hi,

After performing, a fresh install of the project, the home page is starting well and I can navigate with the menu.

But I have an issue while trying to click on `start` or on `save` buttons.

I do not know if I missed something during my last PR #204.

Here is the error message:

```
OperationalError at /

no such table: django_session

Request Method: 	POST
Request URL: 	http://127.0.0.1:8001/
Django Version: 	3.0.4
Exception Type: 	OperationalError
Exception Value: 	

no such table: django_session

Exception Location: 	/mnt/c/Users/Starli0n/Develop/src/github.com/Starli0n/f-firefox-profilemaker/venv/lib/python3.8/site-packages/django/db/backends/sqlite3/base.py in execute, line 396
Python Executable: 	/mnt/c/Users/Starli0n/Develop/src/github.com/Starli0n/f-firefox-profilemaker/venv/bin/python
Python Version: 	3.8.2
Python Path: 	

['/mnt/c/Users/Starli0n/Develop/src/github.com/Starli0n/f-firefox-profilemaker/project',
 '/usr/lib/python38.zip',
 '/usr/lib/python3.8',
 '/usr/lib/python3.8/lib-dynload',
 '/mnt/c/Users/Starli0n/Develop/src/github.com/Starli0n/f-firefox-profilemaker/venv/lib/python3.8/site-packages']

Server time: 	Tue, 15 Sep 2020 10:34:06 +0000
```

Could you please help me on this ?

**Comment by allo-** [comment]
Make sure that the db entries are there (default sqlite should be okay, the app does not use the db itself) and run `./manage.sh migrate`, or set up some other session backend (not recommended, except when you already have something running).

The general steps should be like in the documentation, e.g., the django tutorial or other quick start guides.

**Comment by allo-** [comment]
This may belong into the documentation here as well. Either it should just be "install the app into a django project" or include this step when it is step by step up to runserver.

**Comment by Starli0n** [comment]
Indeed the following command fixes the issue:
```sh
$ ./manage.sh migrate
```

I know about python but I do not know about django at all.

If you do not mind, I could add it into the Install procedure.

**Comment by allo-** [comment]
Fixed in #206 

Thank you for the detailed testing  of what was a bit short in the install docs.
When you add "(fixes #xxx)" to the commit message next time you can automatically close the bug on merge.

**Comment by Starli0n** [comment]
Thank you and I will add "(fixes #xxx)" in the commit message next time :+1:
